docs/TerraToolkitUpgradeGuide-v3.0.0.md

```diff
@@ -0,0 +1,121 @@
+# Terra Toolkit Upgrade Guide v3.0.0
+This document will provide information on upgrading from Terra Toolkit 2.x to 3.0.0.
+
+If you use terra-dev-site please also refer to it's upgrade guide as it abstracts away from additional config that is required here.
```

Comment:
If you use terra-dev-site*,* please also...

Do we have a guide we can link to?